File: Backend/Java env/auth-service/Eclothing/src/main/java/com/example/demo/dto/SellerRegisterRequest.java
package com.example.demo.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;

public class SellerRegisterRequest {
	@NotBlank(message = "Name is required")
	private String name;
	
	@NotBlank(message = "Email is required")
    @Email(message = "Enter valid email")
    private String email;
	
	@NotBlank(message = "Mobile is required")
    private String mobile;
	
	@NotBlank(message = "Password is required")
    private String password;

	@NotBlank(message = "ShopName is required")
    private String shopName;
	
	@NotBlank(message = "GST Number is required")
    private String gstNumber;
	
	@NotBlank(message = "Address is required")
    private String businessAddress;
    
	public SellerRegisterRequest() {
	}
	
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getMobile() {
		return mobile;
	}
	public void setMobile(String mobile) {
		this.mobile = mobile;
	}
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	public String getShopName() {
		return shopName;
	}
	public void setShopName(String shopName) {
		this.shopName = shopName;
	}
	public String getGstNumber() {
		return gstNumber;
	}
	public void setGstNumber(String gstNumber) {
		this.gstNumber = gstNumber;
	}
	public String getBusinessAddress() {
		return businessAddress;
	}
	public void setBusinessAddress(String businessAddress) {
		this.businessAddress = businessAddress;
	}
    
    
}

File: Backend/DotNet env/AuthService/Models/DTOs/SellerRegisterRequest.cs
using System.ComponentModel.DataAnnotations;

namespace AuthService.Models.DTOs
{
    public class SellerRegisterRequest
    {
        [Required(ErrorMessage = "Name is required")]
        public string Name { get; set; } = string.Empty;

        [Required(ErrorMessage = "Email is required")]
        [EmailAddress(ErrorMessage = "Enter valid email")]
        public string Email { get; set; } = string.Empty;

        [Required(ErrorMessage = "Mobile is required")]
        public string Mobile { get; set; } = string.Empty;

        [Required(ErrorMessage = "Password is required")]
        public string Password { get; set; } = string.Empty;

        [Required(ErrorMessage = "ShopName is required")]
        public string ShopName { get; set; } = string.Empty;

        [Required(ErrorMessage = "GST Number is required")]
        public string GstNumber { get; set; } = string.Empty;

        [Required(ErrorMessage = "Address is required")]
        public string BusinessAddress { get; set; } = string.Empty;
    }
}
